### DISCUSSION TITLE
拖拽调整节点顺序，部分位置无法吸附上去

### DISCUSSION THREAD

**Discussion by soom-lupulus** [hizzgdev/jsmind#538] [General]
父节点下有许多子节点，拖动其中一个子节点到最后的位置，此时会吸附到父节点下面的那个同级节点上

这样就无法把其中一个子节点调整到最后的位置。

看了下源码，是不断的更新最近的节点，但是最近的不一定是想吸附连上去的

**Comment by hizzgdev** [comment]
你可以先确认一下你使用的是哪个版本。

在 0.7.5 之前，吸附的逻辑是查找最近的节点，
在 0.7.5 版本时节点吸附的逻辑进行了优化，会优先吸附到“重叠”节点的父节点上。

<img width="405" alt="draggable-node" src="https://github.com/hizzgdev/jsmind/assets/1690290/2f911058-e4e7-4844-8a36-9dea85303ed7">

**Comment by hizzgdev** [comment]
转到 Discussion 区了哈。
